annotation/tests/test_annotations.py

```diff
@@ -53,7 +58,7 @@ def annotator():
 
 
 @pytest.fixture
-def annotation_job(annotator: User, annotation_file: File, categories: Job):
+def annotation_job(annotator, annotation_file, categories):
```

Comment:
If you're using this function in only one place, there's no need to create a fixture.

Reply:
Add types.